src/common/recordbatch/src/lib.rs

```diff
@@ -132,6 +134,21 @@ impl RecordBatches {
             index: 0,
         })
     }
+
+    pub fn into_df_stream(
```

Comment:
Looks like this method won't fail anyway, simplify return type to `DfSendableRecordBatchStream`?

Reply:
nice catch